client/src/components/DemoSelect/DemoSelect.tsx

```diff
@@ -0,0 +1,48 @@
+import { Chip, Divider, Typography } from '@material-ui/core';
+import useStyles from './useStyles';
+import PetsIcon from '@material-ui/icons/Pets';
+import HouseIcon from '@material-ui/icons/House';
+
+interface Props {
+  label: string;
+  user: ({ username, email, password }: { username: string; email: string; password: string }) => void;
+}
+
+interface DemoUser {
+  username: string;
+  email: string;
+  password: string;
+}
+
+const mockOwnerUser = {
+  username: 'Dog Owner Demo',
+  email: 'dogownerdemo@gmail.com',
+  password: 'dogownerdemo',
+};
+
+const mockSitterUser = {
+  username: 'Dog Sitter Demo',
+  email: 'dogsitterdemo@gmail.com',
+  password: 'dogsitterdemo',
+};
+
+function Demo({ label, user }: Props): JSX.Element {
+  const classes = useStyles();
+
+  function setUserDemo({ username, email, password }: DemoUser) {
+    user({ username, email, password });
+  }
+
+  return (
+    <>
+      <Divider variant="middle" />
+      <Typography className={classes.header} align="center" color="textPrimary">
+        {label}
+      </Typography>
+      <Chip label="Owner" icon={<PetsIcon />} onClick={() => setUserDemo(mockOwnerUser)} className={classes.demo} />
```

Comment:
As opposed to setting the state in the parent component, why don't we call the login function here itself? 😄

Reply:
As the code is just now, my demo button is just populating the forms for Log In and Sign Up so the user can click the Login and Create button respectively after that.  Those buttons keep maintaining the responsibility of the login.   Maybe it's over complicated 😅 I'll change it.